bats/tests/k8s/helm-install-rancher.bats

```diff
@@ -98,22 +163,65 @@ deploy_rancher() {
     helm upgrade \
         --install cert-manager jetstack/cert-manager \
         --namespace cert-manager \
-        --set installCRDs=true \
+        --set crds.enabled=true \
+        --set crds.keep=true \
+        --set prometheus.enabled=false \
         --set "extraArgs[0]=--enable-certificate-owner-ref=true" \
         --create-namespace
+    try assert_not_empty_list helm list --namespace cert-manager --deployed --output json --selector name=cert-manager
+    wait_for_kube_deployment_available --namespace cert-manager cert-manager
 
     local host
     host=$(traefik_hostname) || return
 
     comment "Installing rancher $rancher_chart_version"
+    # The helm install can take a long time, especially on CI
```

Comment:
I think this comment should spell out that we intentionally don't use `--wait` and `--timeout` options for `helm`, but instead manually check progress along the way because things are so slow in CI.